components/StyledButton.js

```diff
@@ -15,4 +12,14 @@ export const StyledButton = styled.button`
   &:active {
     box-shadow: 2px 2px 5px indigo;
   }
+  &.generate {
+    position: absolute;
+    bottom: 0.5rem;
+    right: 0.5rem;
+  }
+  &.variations {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
```

Comment:
could be named "center", which is closer to what this variation actually does.

you should also work with props here for consistency (see example above)